library/upstart_service.py

```diff
@@ -58,8 +58,8 @@ def main():
             cmd=dict(default=None, required=True),
             args=dict(default=None),
             user=dict(default=None, required=True),
-            config_dirs=dict(default=None),
-            config_files=dict(default=None),
+            config_dirs=dict(default=None, type='list'),
```

Comment:
config_dirs expects a comma separated string, and does its own splitting inside. Same with config_files. Probably don't want to change the module behavior at this point.